ecms_base/features/custom/ecms_emergency_notification/config/install/field.field.node.emergency_notification.field_emergency_weight.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_emergency_weight
+    - node.type.emergency_notification
+  module:
+    - options
+id: node.emergency_notification.field_emergency_weight
+field_name: field_emergency_weight
+entity_type: node
+bundle: emergency_notification
+label: Weight
+description: 'Add a sorting weight to the notification. 9 being the highest priority and appearing first in the list. -9 being the lowest priority and appearing last.'
```

Comment:
Drupal generally goes the other way with weights. Bigger numbers `sink`, while lighter numbers `float`. It might make sense to follow the same logic for consistency.

Reply:
This follows the same pattern as the weight in the Notifications content type.